### DISCUSSION TITLE
classes/class_multiplayerapi

### DISCUSSION THREAD

**Discussion by giscus** [godotengine/godot-docs-user-notes#136] [User-contributed notes]
# classes/class_multiplayerapi

Inherits: RefCounted< Object Inherited By: MultiplayerAPIExtension, SceneMultiplayer High-level multiplayer API interface. Description: Base class for high-level multiplayer API implementations. Se...

https://docs.godotengine.org/en/stable/classes/class_multiplayerapi.html

**Comment by cheesebellies** [comment]
If you've arrived here in search of a way to send basic packets instead of using an RPC call, try the following links:
[SceneMultiplayer](https://docs.godotengine.org/en/stable/classes/class_scenemultiplayer.html#class-scenemultiplayer-method-send-bytes) (inherited by this class, just use `multiplayer.send_bytes()`)
[PacketPeer](https://docs.godotengine.org/en/stable/classes/class_packetpeer.html#class-packetpeer-method-get-packet) (Inherited by all MultiplayerPeer implementations)
